### PR TITLE
210 reveal

### PR DESCRIPTION
**Pull request by eneo5541**

### CODE REVIEW

File: src/components/ux-reveal/ux-reveal.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class="ux-reveal">
+
+    <a target="_blank" on-click="toggleModal" class="open-reveal-modal {{class}}">{{text}}</a>
+
+    <div class="reveal-modal-bg {{#if modalVisible}}modal-visible{{/if}}">
+        <div class="reveal-modal {{#if modalVisible}}modal-visible{{/if}}">
+            {{#if isDataModel}}
+                {{{payload}}}
```
Comment (ivanwills) [inline src/components/ux-reveal/ux-reveal.hbs]: 'content' is more our standard rather than 'payload'

File: src/components/ux-reveal/ux-reveal.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	},
+
+	oninit: function () {
+
+		this.on('toggleModal', function (e) {
+			var visible = this.get('modalVisible');
+			this.set('modalVisible', !visible);
+
+			this.fire('toggleReveal', {
+				visible: this.get('modalVisible')
```
Comment (ivanwills) [inline src/components/ux-reveal/ux-reveal.js]: shouldn't you just use the visible variable use before rather than get it again?

**Comment by eneo5541** [comment]
Tidying up ux-reveal, as per Ivan's suggestions.

File: src/components/ux-reveal/ux-reveal.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	},
+
+	oninit: function () {
+
+		this.on('toggleModal', function (e) {
+			var visible = this.get('modalVisible');
+			this.set('modalVisible', !visible);
+
+			this.fire('toggleReveal', {
+				visible: visible
```
Comment (ivanwills) [inline src/components/ux-reveal/ux-reveal.js]: should that be `visible` or `!visible`?

**Pull request merged**